packages/camera/camera_android_camerax/pigeons/camerax_library.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(
+  PigeonOptions(
+    dartOut: 'lib/src/camerax.pigeon.dart',
+    dartTestOut: 'test/test_camerax.pigeon.dart',
+    dartOptions: DartOptions(copyrightHeader: <String>[
+      'Copyright 2013 The Flutter Authors. All rights reserved.',
+      'Use of this source code is governed by a BSD-style license that can be',
+      'found in the LICENSE file.',
+    ]),
+    javaOut:
+        'android/src/main/java/io/flutter/plugins/camerax/GeneratedCameraXLibrary.java',
+    javaOptions: JavaOptions(
+      package: 'io.flutter.plugins.camerax',
+      className: 'GeneratedCameraXLibrary',
+      copyrightHeader: <String>[
+        'Copyright 2013 The Flutter Authors. All rights reserved.',
+        'Use of this source code is governed by a BSD-style license that can be',
+        'found in the LICENSE file.',
+      ],
+    ),
+  ),
+)
+@HostApi()
```

Comment:
This and the other class should include the test api: 
```suggestion
@HostApi(dartHostTestHandler: 'TestJavaObjectHostApi')
```